src/zenml/__init__.py

```diff
@@ -23,3 +23,8 @@
 from zenml.logger import init_logging  # noqa
 
 init_logging()
+
+# The following code is needed for `zenml.hub` subpackages to be found
+from pkgutil import extend_path
+
+__path__ = extend_path(__path__, __name__)
```

Comment:
Is this necessary both here and in the `hub/__init__.py`?

Reply:
apparently yes but don't ask me why 😄

Reply:
Why?

Reply:
Well because that is how package resolving works in Python, see [the offical Python docs]( https://packaging.python.org/en/latest/guides/packaging-namespace-packages/) for reference

Reply:
Hahah 😄

Reply:
![Alt Text](https://media.tenor.com/oyVcC9YdDukAAAAC/well-damn.gif)